data_relay.py

```diff
@@ -51,82 +47,158 @@
-            ('f', 'altitude_ki'),
-            ('f', 'throttle_kd'),
-            ('f', 'throttle_kp'),
-            ('f', 'throttle_ki'),
-            ('f', 'flap_kd'),
-            ('f', 'flap_kp'),
-            ('f', 'flap_ki'),
-            ('f', 'path_gain'),
-            ('f', 'orbit_gain'),
-            ('h', 'autonomousLevel'),
-            ('h', 'startup_error_codes'),
-            ('h', 'startupSettings'),
-            ('h', 'dl_transmission_errors'),
-            ('h', 'ul_receive_errors'),
-            ('B', 'ul_rssi'),
-            ('B', 'uhf_rssi'),
-            ('B', 'uhf_link_quality'),
+            ('f', 'ground_speed_kp'),
+            ('f', 'path_kp'),
+            ('f', 'orbit_kp')
+            ),
+        PACKET_TYPE_CHANNELS: ( #34 bytes + 48 bytes padding
+            ('h', 'ch1_in'),
+            ('h', 'ch2_in'),
+            ('h', 'ch3_in'),
+            ('h', 'ch4_in'),
+            ('h', 'ch5_in'),
+            ('h', 'ch6_in'),
+            ('h', 'ch7_in'),
+            ('h', 'ch8_in'),
+            ('h', 'ch1_out'),
+            ('h', 'ch2_out'),
+            ('h', 'ch3_out'),
+            ('h', 'ch4_out'),
+            ('h', 'ch5_out'),
+            ('h', 'ch6_out'),
+            ('h', 'ch7_out'),
+            ('h', 'ch8_out'),
+            ('B', 'channels_scaled'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
+            ('x', 'one byte of padding'),
```

Comment:
This can still be made cleaner. Instead of sequences of identical lines, you can make a few changes so the packets are appended with `18x` or something.